### PR TITLE
Notifications: Fixing Table Section Separators

### LINKED ISSUE #4097: Notifications: Section Header with incorrect separators width
#### Steps:
1. To ease things out, hack [this method](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m#L202), so that the ratings view is always rendered
2. Launch WPiOS
3. Switch to the Notifications Tab

Shortly after, the Ratings View should be onscreen. Notice that the Section Headers separator's width is incorrect (in the screenshot below, `Today` and `Yesterday` sections).

ios simulator screen shot jul 31 2015 12 47 31 pm

### PR DESCRIPTION
**Pull request by jleandroperez**
#### Steps:
1. To ease things out, hack [this method](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.m#L202), so that the ratings view is always rendered
2. Launch WPiOS
3. Switch to the Notifications Tab

Few seconds afterwards, the Ratings View should be onscreen. Note that the Section Headers should be displayed correctly, and the top and bottom separator's width should match with the cell's width.

Needs Review: @astralbodies (Thanks Aaron!)

Closes #4097

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Notifications/Views/NoteTableHeaderView.swift [hunk 1]
```diff
@@ -1,16 +1,23 @@
...
 {
     // MARK: - Public Properties
     public var title: String? {
         set {
             // For layout reasons, we need to ensure that the titleLabel uses an exact Paragraph Height!
-            if let unwrappedTitle = newValue?.uppercaseString {
-                titleLabel.attributedText = NSAttributedString(string: unwrappedTitle, attributes: Style.sectionHeaderRegularStyle)
-                setNeedsLayout()
-            }
+            let unwrappedTitle = newValue?.uppercaseString ?? String()
```
Comment (astralbodies) [inline WordPress/Classes/ViewRelated/Notifications/Views/NoteTableHeaderView.swift]: Make sure to use uppercaseStringWithLocale(_:)

**Comment by jleandroperez** [comment]
@astralbodies Switched over to `uppercaseStringWithLocale`. Thank you!

**Comment by astralbodies** [comment]
Awesome! Let me check out the code and run it to give you the final okay.

**Comment by astralbodies** [comment]
:shipit:

**Comment by jleandroperez** [comment]
Thank you sir!!

**Pull request merged**